Makefile

```diff
@@ -18,6 +18,7 @@ DESTDIR ?= /
 PREFIX ?= /usr/local
 SYSCONF ?= etc
 INSTALL_DIR = $(PREFIX)/share
+# RUN_DIR ?= /run
```

Comment:
Should this remain commented?

Reply:
Good point. No, it should not be commented. When somebody tries to use make without arguments, then some default value should be defined in `Makefile`.